articles/flows/concepts/auth-code-pkce.md

```diff
@@ -47,13 +53,19 @@ Because the PKCE-enhanced Authorization Code Flow builds upon the [standard Aut
 
 ## How to implement it
```

Comment:
The workflow above still mentions Mobile Apps explicitly... Step 1 might be best as just Application. (same with the diagram)